src/stylus/react-selectize.styl

```diff
@@ -0,0 +1,929 @@
+.example .react-selectize.multi-select .removeable-emoji {
```

Comment:
Cool! I've gotten that bundled now, and the dropdowns work without this ugly file